File: ios/RNHelloDoctor/Video/HDVideoRemoteViewManager.m
//
//  HDVideoRemoteViewManager.m
//  hellodoctor
//
//  Created by HelloDoctor on 9/12/20.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(HDVideoRemoteViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(participantSID, NSString)

@end

File: ios/RNHelloDoctor/Video/HDVideoRemoteViewManager.swift
//
//  HDVideoLocalViewManager.swift
//  RNHelloDoctor
//
//  Created by HelloDoctor on 3/29/22.
//

import Foundation
import TwilioVideo
import UIKit

@objc(HDVideoRemoteViewManager)
class HDVideoRemoteViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }

    override func view() -> UIView! {
        return HDVideoView()
    }
}
